src/DwellingImporter.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace NMF;
+
+class DwellingImporter{
+    private $db;
+    private $api;
+    
+    /**
+     * class to import the data from the API into the database
+     *
+     * @param \PDO $db PDO connection to the db
+     * @param Object $api instance of dwelling api
+     */
+    public function __construct(\PDO $db,Object $api)
+    {
+        $this->db = $db;
+        $this->api = $api;
+    }
+    /** 
+     * deletes all data from the dwellings, statuses and types tables 
+     */
+    private function deleteTables()
+    {
+        $query = $this->db->prepare("TRUNCATE TABLE dwellings;");
```

Comment:
Should really be TRUNCATE TABLE `dwellings`

Reply:
GitHub's formatting rules hate me, I was trying to say there should be backticks around dwellings